README.md

```diff
@@ -56,27 +56,99 @@ Code quality:
     ```
 
 1. Start app with required services
-    ```shell script
+    ```shell
     make runserver
     ```
 
 1. Service will be available at `http://127.0.0.1:8042/`
 
-## System requires
+## Start the project for development without Docker
+1. Clone the repo and go to the project directory
+    ```shell
+    git clone git@github.com:Uma-Tech/parrot.git && \
+    cd parrot
+    ```
+
+1. Install dependencies
+    ```shell
+    poetry install
+    ```
+
+1. Start databases  
+    _You can start databases in any other way_
+   ```shell
+    # start postgres
+   docker run \
+       --name postgres \
+       -d \
+       --rm \
+       -e POSTGRES_PASSWORD=parrot \
+       -e POSTGRES_USER=parrot \
+       --network host \
+       -v parrot_db:/var/lib/postgresql/data \
+       postgres
+   
+    # start redis
+   docker run \
+       --name redis \
+       -d \
+       --rm \
+       --network host \
+       redis 
+    ```
+
+1. Setting require environment variables  
+   _Alternative, you can create a local `.env` file with the variables_
+    ```shell
+    export PARROT_DB_HOST=127.0.0.1
```

Comment:
```suggestion
    PARROT_DB_HOST=127.0.0.1
```

I think export is not used in the `.env` files, it's a bash syntax.

Reply:
These are bash commands, not the .env file